Next.XP 2.0.0 release (#678)
enonic/nextxp-demo: docs/parts.adoc

+
. **Add parts to the page**
+
Back in Content Studio, edit the `/hmdb/persons` folder item.
Expand the page preview section by clicking display icon in the top right corner and select `Main page` controller in the dropdown in the middle of page.

> Create the page and place the new part into the region (`right-click -> Insert`, or drag-n-drop from the `Insert panel`` on the right hand side).

Reviewer: ```suggestion
Create the page and add a new part to the region (`right-click -> Insert`, or drag-n-drop from the `Insert panel`` on the right hand side).
```
